src/Snowcap/Emarsys/Client.php

```diff
@@ -264,6 +264,24 @@ public function updateContact(array $data)
     }
 
     /**
+     * Updates one or more contacts/recipients, identified by an external ID. If the contact does not exist in the
+     * database, it is created.
+     *
+     * @param array $data
```

Comment:
Should be `mixed[]` I guess. Does not help much but at least shows that developer thought about it. 😊